### PR TITLE
[CA-1888] Update raven-logback to 8.0.3 in develop (DO NOT MERGE YET)

### PR DESCRIPTION
**Pull request by broadbot**
Updates com.getsentry.raven:raven-logback from 7.8.6 to 8.0.3.

I'll automatically update this PR to resolve conflicts as long as you don't change it yourself.

If you'd like to skip this version, you can just close this PR. If you have any feedback, just mention me in the comments below.

Configure Scala Steward for your repository with a [`.scala-steward.conf`](https://github.com/scala-steward-org/scala-steward/blob/17235ca1e60ff61e35fe7b7cf51163a15ffc435d/docs/repo-specific-configuration.md) file.

Have a fantastic day writing Scala!

<details>
<summary>Ignore future updates</summary>

Add this to your `.scala-steward.conf` file to ignore future updates of this dependency:
```
updates.ignore = [ { groupId = "com.getsentry.raven", artifactId = "raven-logback" } ]
```
</details>

labels: library-update, early-semver-major, semver-spec-major, commit-count:1

### CODE REVIEW

**Comment by andy7i** [comment]
jenkins retest -- test results no longer visible

**Comment by andy7i** [comment]
jenkins retest - fiab-start should work now

**Comment by andy7i** [comment]
jenkins retest
one test failed due to a 5 second timeout https://fc-jenkins.dsp-techops.broadinstitute.org/job/sam-fiab-test-runner/41549/consoleFull

**Review by ehigham — APPROVED** [review]
8.0.3 seems to have the same vulnerabilities as the previous version so I guess we're not introducing anything worse with this https://mvnrepository.com/artifact/com.getsentry.raven/raven-logback/8.0.3
Provided the tests pass, ship it!

File: project/Dependencies.scala [hunk 1]
```diff
@@ -33,7 +33,7 @@ object Dependencies {
 
   val logstashLogback: ModuleID = "net.logstash.logback"      % "logstash-logback-encoder" % "6.6"
   val logbackClassic: ModuleID = "ch.qos.logback"             %  "logback-classic" % "1.2.10"
-  val ravenLogback: ModuleID =   "com.getsentry.raven"        %  "raven-logback"   % "7.8.6"
+  val ravenLogback: ModuleID =   "com.getsentry.raven"        %  "raven-logback"   % "8.0.3"
```
Comment (gpolumbo-broad) [inline project/Dependencies.scala:36]: https://docs.sentry.io/platforms/java/guides/logback/legacy/migration/

We can make the version upgrade as described on the PR, that's fine, but I'm wondering if we shouldn't also make the changes described here?  I'm worried that if 8.0.x is the final raven-* version, will scala steward be smart enough to know that the overall package name changed for future updates?  I suspect not.

**Review by gpolumbo-broad — COMMENTED** [review]
I'm not sure how many other changes we may need to make with this to make it work properly with the new packaged version of the lib.  Id be ok holding off on this PR and get the other easy scala steward ones done first if that's how we want to do it.

File: project/Dependencies.scala (hunk 1 above)
Comment (andy7i) [reply project/Dependencies.scala:36]: good catch! did you already know about the raven -> sentry name change or were your spidey senses tingling?

**Review by andy7i — CHANGES_REQUESTED** [review]
update raven -> sentry

**Comment by gpolumbo-broad** [comment]
No spidey senses.  Generally if I see a minor version bump that isn't too large or just a patch version, I just thumb and move on.  If it's a major version bump though, it's worth googling and checking out the change logs.  Semantic Versioning for the win!  When I googled in this case, it was the first thing that came up.  So as we go through the rest of these scala steward tickets, if there are any other major version bumps, make sure you go check out the changelogs for those.

**Pull request closed without merging**